vault-client/lx/init.sls

```diff
@@ -0,0 +1,36 @@
+# Python2
+{%- if salt.grains.get('pythonversion')[0] | int == 2 %}
+install_pip_module:
```

Comment:
This was ok when we were embedding this formula in another project, but as a standalone formula messing with the pip install is really tricky. For now, I'd rather defer this setup to the user. Let them provide a working pip install as a prereq to using this formula.

That means this sls can be reduced to:

```
install_python_dependencies:
  pip.installed:
    - pkgs:
      - hvac
{%- if salt.grains.get('pythonversion')[0] | int == 3 %}
    - target: /usr/lib/python3.6/site-packages
{%- endif %} 
    - reload_modules: True
    - ignore_installed: True
```